src/components/Error/Error.js

```diff
@@ -0,0 +1,14 @@
+import './Error.css'
+import shockedCat from '../../images/hang-niu-Tn8DLxwuDMA-unsplash.jpg'
+
+function Error({error}) {
+  return (
+    <section className='error'>
+      <h2>{error}</h2>
+      <img src={shockedCat}/>
```

Comment:
We should probs give an alt text to this image for accessibility!

Reply:
Good catch!